src/v2/Apps/getAppRoutes.tsx

```diff
@@ -32,9 +32,9 @@ export function getAppRoutes(): RouteConfig[] {
     {
       routes: artworkRoutes,
     },
-    {
-      routes: collectRoutes,
-    },
+    // {
```

Comment:
(nonblocking) Might be worth adding a comment here about whats happening (though it might also be self evident since we'll be moving stuff over pretty fast)